packages/react-components/react-list-preview/src/components/List/List.tsx

```diff
@@ -5,19 +5,16 @@ import { useList_unstable } from './useList';
 import { renderList_unstable } from './renderList';
 import { useListStyles_unstable } from './useListStyles.styles';
 import type { ListProps } from './List.types';
+import { useListContextValues_unstable } from './useListContextValues';
 
-/**
- * List component - TODO: add more docs
- */
 export const List: ForwardRefComponent<ListProps> = React.forwardRef((props, ref) => {
   const state = useList_unstable(props, ref);
+  const listContext = useListContextValues_unstable(state);
```

Comment:
```suggestion
  const contextValues = useListContextValues_unstable(state);
```

nit: Let's follow naming patterns like in other components